example-process-runtime-bundle-service/README.md

```diff
@@ -26,6 +26,7 @@ docker run -it -p 127.0.0.1:8080:8080 \
   --env SPRING_RABBITMQ_USERNAME=*** \
   --env SPRING_RABBITMQ_PASSWORD=*** \
   --env SPRING_RABBITMQ_PORT=5672 \
+  --env TRIGGERS_PATH=/root/.activiti/connector-exchange-volume/1/triggers \
```

Comment:
no sorry i forgot to mention that you need to apply the same strategy done for processes env variables above
and add to the dockerfile the trigger models like done for processes in /maven/processes. That would be /maven/triggers i believe

Reply:
this is the file to add the triggers model: https://github.com/Alfresco/example-process-application/blob/master/example-process-runtime-bundle-service/Dockerfile#L3

In the readme i would also add that the variable for trigger is optional